src/Containers/Catagories/VideoPlayer/Notes/Notes.js

```diff
@@ -0,0 +1,28 @@
+import classes from './Notes.module.css';
+import {useContext} from 'react'
+import {AuthContext} from '../../../../Store/AuthReducer'
+import {CatagoriesContext} from '../../../../Store/CatagoriesReducer'
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome'
+import {faTrash} from '@fortawesome/free-solid-svg-icons'
+
+const Notes=({name,note,id,noteId,videoId})=>{
+    const {userId}=useContext(AuthContext);
+    const {deleteNote}=useContext(CatagoriesContext)
```

Comment:
we can make this a custom hook "useCategories"

Reply:
Thanks for the feedback danish! I'll refactor the code with custom hooks